Formula/wine.rb

```diff
@@ -134,6 +478,16 @@ def caveats
     s
   end
 
+  def post_install
+    # For fontconfig
+    ohai "Regenerating font cache, this may take a while"
+    system "#{prefix}/deps/bin/fc-cache", "-frv"
```

Comment:
Yes, it needs to be done before programs depending on `fontconfig` (i.e. wine) run for the first time.